mostStonesRemovedWithSameRowOrColumn.js

```diff
@@ -0,0 +1,121 @@
+/*
+On a 2D plane, we place n stones at some integer coordinate points. Each coordinate point may have at most one stone.
+
+A stone can be removed if it shares either the same row or the same column as another stone that has not been removed.
+
+Given an array stones of length n where stones[i] = [xi, yi] represents the location of the ith stone, 
+return the largest possible number of stones that can be removed.
+*/
+
+//
+// [0, 0],
+// [0, 1],
+// [1, 0],
+// [1, 2],
+// [2, 1],
+// [2, 2],
+
+const { Queue } = require('./utils/queue');
+
+const X_IDX = 0;
+const Y_IDX = 1;
+
+const getCoordinateString = (x, y) => `${x}, ${y}`;
+
+const getGroupedCoordinates = (nodeCoordinates, IDX) => {
```

Comment:
All caps naming should be reserved for constants (like the ones you're declaring at the top of the file). This should be lowercased `idx`.